doc/releases/release-notes-3.2.rst

```diff
@@ -364,9 +428,51 @@ Libraries / Subsystems
   * Added :c:macro:`LOG_RAW` for logging strings without additional formatting.
     It is similar to :c:macro:`LOG_PRINTK` but do not append ``<cr>`` when new line is found.
 
+* LoRaWAN
+
+* Modbus
+
+* Power management
+
+* RTIO
+
+* SD Subsystem
+
+* Settings
+
+* Shell
+
+* Storage
+
+* Testsuite
+
+* Tracing
+
 HALs
 ****
 
+* Atmel
+
+* Espressif
+
+* GigaDevice
+
+* Nordic
+
+* NXP
+
+* Renesas
+
+* RPi Pico
+
+* Silabs
+
+* STM32
+
+* Wurth Elektronik
+
+* Xtensa
```

Comment:
what about:

* Altera
* Cypress
* Infineon
* Microchip
* Nuvoton
* OpenISA
* QuickLogic
* ST
* Telink
* TI

Reply:
I had a look at changes in west.yaml for v3.1.0..main diff and only added the ones who changed. Though I double checked it now and it looks like ST Telink and TI should be there. Let me add those.

Reply:
Done, just added those three.